lib/msal-core/src/ServerRequestParameters.ts

```diff
@@ -45,10 +46,11 @@ export class ServerRequestParameters {
    * @param redirectUri
    * @param state
    */
-  constructor (authority: Authority, clientId: string, scope: Array<string>, responseType: string, redirectUri: string, state: string ) {
+  constructor (authority: Authority, clientId: string, scope: Array<string>, responseType: string, redirectUri: string, state: string, popToken?: string ) {
```

Comment:
would be nice to eventually refactor this to take options even on required fields 


new ServerRequestParameters({
   authority,
   clientId,
   .....
   popToken: 'fdav32r4raacd4rt4a......'
})

Reply:
not for this pr